### PR TITLE
Responsive designs

### PR DESCRIPTION
**Pull request by Elkrival**
Added responsive stylings to app

Screenshot 2023-11-30 at 3 18 37 PM
Screenshot 2023-11-30 at 3 19 12 PM
Screenshot 2023-12-01 at 7 51 57 PM
Screenshot 2023-12-01 at 7 52 05 PM
Screenshot 2023-12-04 at 8 40 41 AM

### CODE REVIEW

File: constants/vars.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export const N_A = 'N/A'
 export const TOTAL_LABEL = 'Total'
 export const TOTALS = 'Totals'
-export const DISALLOWED_STUDIES = ['combined', 'files']
+export const DISALLOWED_STUDIES = ['combined', 'files', 'ProNET']
```
Comment (JackVCurtis) [inline constants/vars.js:4]: ? How is this related? Random bug fix or something?

**Comment by JackVCurtis** [comment]
The `sx` stuff still feels weird to me but 🤷 . Other than that one comment on adding ProNet to DISALLOWED_STUDIES this looks great!

File: constants/vars.js (hunk 1 above)
Comment (Elkrival) [reply constants/vars.js:4]: The ProNet site is made up and this ensures that this data is not included in the charts results

**Comment by Elkrival** [comment]
> The `sx` stuff still feels weird to me but 🤷 . Other than that one comment on adding ProNet to DISALLOWED_STUDIES this looks great!

I feel the same way, it definitely drives the styling for the individual component, but i'm not a fan of their made up properties like p or mt, etc...

File: constants/vars.js (hunk 1 above)
Comment (JackVCurtis) [reply constants/vars.js:4]: So unrelated bug fix?

**Review by JackVCurtis — APPROVED** [review]
👍  I'm working on AWS so this won't auto deploy, but looks great!

File: constants/vars.js (hunk 1 above)
Comment (Elkrival) [reply constants/vars.js:4]: yes
